numpy/core/src/umath/simd.inc.src

```diff
@@ -3697,4 +3736,142 @@ sse2_@kind@_BOOL(@type@ * op, @type@ * ip, const npy_intp n)
 
 #endif /* NPY_HAVE_SSE2_INTRINSICS */
 
+#ifdef NPY_HAVE_NEON
+/**begin repeat
+ *  #type = npy_float, npy_double#
+ *  #TYPE = FLOAT, DOUBLE#
+ *  #scalarf = npy_sqrtf, npy_sqrt#
+ *  #Nvtype128 = float32x4_t, float64x2_t#
+ *  #Nvtype64 = float32x2_t, float64x1_t#
+ *  #Nvpre = v, v#
+ *  #Nvsuf = f32, f64#
+ */
+
+
+/**begin repeat1
+* Arithmetic
+* # kind = add, subtract, multiply, divide#
+* # OP = +, -, *, /#
+* # VOP = add, sub, mul, div#
+*/
+
+static void
+neon_binary_@kind@_@TYPE@(@type@ * op, @type@ * ip1, @type@ * ip2, npy_intp n)
+{
+#ifdef  __aarch64__
+    const npy_intp vector_size_bytes = 16;
```

Comment:
Can you use the common vector_size_bytes that was defined above? I suggest a macro define.

Reply:
defined macro vector_size_bytes=16 (8) universally for __aarch64__ (__aarch32__) at the very beginning.